netmap/netmap_contract.go

```diff
@@ -94,6 +94,24 @@ func _deploy(data interface{}, isUpdate bool) {
 
 	if isUpdate {
 		common.CheckVersion(args.version)
+
+		prefix := []byte(snapshotKeyPrefix)
+		for i := 0; i < DefaultSnapshotCount; i++ {
```

Comment:
Shouldn't we read snapshot count from `snapshotCountKey`?

Reply:
Fixed